README.md

```diff
@@ -1,20 +1,38 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
-
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```

Comment:
Not your changes

Reply:
@Mnickii I've removed the changes that are not mine. I'm working on solving the merge conflict now.